doc/whats_new/v0.24.rst

```diff
@@ -70,6 +70,13 @@ Changelog
   `init_size_`, are deprecated and will be removed in 0.26. :pr:`17864` by
   :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :class:`cluster.AgglomerativeClustering` has now an attribute
+  `distances_` if `compute_distances` is set to `True`, allowing to make
```

Comment:
`distances_` can also exist even when it's set to False...

Maybe, something along the lines of ,
```
|Fix| Added a `compute_distances` parameter to :class: .. When True distances are computed and stored in the `distances_` attribute even when no distance_threshold is used. This can be
        used to make dendrogram visualization, but has impact on efficiency due
        to the additional computational cost and used memory.
```
?

Reply:
Fixed in 61e5c5a.